recipes/tbb/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.7
+
+make
+make test
+
+install -d ${PREFIX}/lib
+# filter libtbb.dylib ( or .so ), libtbbmalloc.dylib ( or .so )
+cp `find . -name "*lib*" | grep tbb | grep release` ${PREFIX}/lib
```

Comment:
Maybe one day.... See this [thread]( https://software.intel.com/en-us/forums/intel-threading-building-blocks/topic/347428 ). 😢